libs/output-mapping/src/DeferredTasks/LoadTableQueue.php

```diff
@@ -33,7 +34,15 @@ public function __construct(Client $client, array $loadTableTasks)
     public function start()
     {
         foreach ($this->loadTableTasks as $loadTableTask) {
-            $loadTableTask->startImport($this->client);
+            try {
+                $loadTableTask->startImport($this->client);
+            } catch (ClientException $e) {
```

Comment:
Podle me by se ty chyby mely handlovat tady a ne primo v tech tascich.
Chyby z Storage API jobu se taky handluji v tehle classe ve `waitForAll`. Takze alespon to je pohromade.

Reply:
JJ, to je lepší takhle